errtracker/errtracker.go

```diff
@@ -42,7 +42,13 @@ var (
 	CrashDbURLBase string
 	SnapdVersion   string
 
-	machineID       = "/var/lib/dbus/machine-id"
+	// The machine-id file is at different locations depending on how the system
+	// is setup. On Fedora for example /var/lib/dbus/machine-id doesn't exist
+	// but we have /etc/machine-id. See
+	// https://www.freedesktop.org/software/systemd/man/machine-id.html for a
+	// few more details.
+	machineIDs       = []string{"/var/lib/dbus/machine-id", "/etc/machine-id"}
```

Comment:
We could alternatively default to read /etc/machine-id on all systems but I am not sure why /var/lib/dbus/machine-id was preferred over /etc/machine-id. Maybe because of 14.04 support?